cmd/fyne/commands/package-darwin.go

```diff
@@ -25,7 +27,10 @@ func (p *packager) packageDarwin() error {
 	info := filepath.Join(contentsDir, "Info.plist")
 	infoFile, _ := os.Create(info)
 
-	tplData := darwinData{Name: p.name, ExeName: exeName, AppID: p.appID}
+	tplData := darwinData{
+		Name: p.name, ExeName: exeName, AppID: p.appID,
+		Version: p.appVersion, Build: p.appBuild,
```

Comment:
Please use either single line, or multi line, instead of this hybrid.

Reply:
When new things like this come up that are not failures in the lint command we should really capture the requirement for our contributor documentation.

Reply:
Good idea. Though there is a limit to how many things should be explicit for this project, vs how many are common Go idioms - eg we don't explicitly say not to end lines with semi-colon, even tho it is required by the language's formal grammar - luckily Go is nice enough to add missing semi-colons for us (https://medium.com/golangspec/automatic-semicolon-insertion-in-go-1990338f2649)

Reply:
It's not really the same as goimports will remove the trailing semicolon automatically.

If it is important enough to block a PR then I think it should be captured through the automatic tools, through Go docs or in our specs

Reply:
Added to https://github.com/fyne-io/fyne/wiki/Contributing, feel free to tweak